src/puptoo/process.py

```diff
@@ -123,9 +123,7 @@ def system_profile(
         profile["cores_per_socket"] = lscpu.info['Cores per socket']
 
     if sap:
-        profile["tags"].append({"namespace": "insights-client",
-                                "key": "sap_system",
-                                "value": "True"})
+        profile["sap_system"] = "True"
```

Comment:
wouldn't a boolean be better?

Reply:
Yes.

Reply:
Also, would it make sense to set it to False if it's not an SAP system? Or do we not know what reliably enough?

Reply:
It's possible that the data we use to determine won't arrive because a customer chose not to send it.  So I don't think we can confidently say `False`.